api/v1/controllers/AuthController.js

```diff
@@ -1,5 +1,8 @@
+var bodyParser = require('body-parser');
 var Promise = require('bluebird');
```

Comment:
Either we should change this `Promise` to `_Promise` or the other way around. See [the previous note](https://github.com/HackIllinois/api-2017/pull/21/files#diff-ed604b73816be1981ea75be666f27990R3).